nautobot/ipam/models.py

```diff
@@ -116,6 +117,12 @@ class VRF(PrimaryModel):
         through="ipam.VRFDeviceAssignment",
         through_fields=("vrf", "device"),
     )
+    virtual_machines = models.ManyToManyField(
```

Comment:
Should `devices` and `virtual_machines` be `blank=True` for form validation purposes? Or is that not necessary?

Reply:
They appear to be working fine from the device edit view and vrf edit view